### ISSUE TITLE
[spec failure] standard_couch_with_expires overwrites existing values

### ISSUE THREAD

**Issue by asppsa** [minad/moneta#126]
~~~
     Failure/Error: raise "HTTP error #{response.status}" unless response.status == 201
     RuntimeError:
       failure: 12 tests, on:
       [2290273571719036605, false, 0.9153576264542762]
       HTTP error 409
     Shared Example Group: :store called from ./spec/helper.rb:241
     # ./lib/moneta/adapters/couch.rb:53:in `store'
     # ./lib/moneta/transformer.rb:135:in `store'
     # ./lib/moneta/proxy.rb:62:in `store'
     # ./lib/moneta/expires.rb:39:in `store'
     # ./lib/moneta/proxy.rb:62:in `store'
     # ./lib/moneta/wrapper.rb:17:in `block in store'
     # ./lib/moneta/logger.rb:66:in `wrap'
     # ./lib/moneta/wrapper.rb:17:in `store'
     # ./lib/moneta/mixins.rb:188:in `[]='
     # ./spec/features/store.rb:49:in `block (3 levels) in <top (required)>'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/property.rb:33:in `call'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/property.rb:33:in `block in check'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/generator.rb:83:in `call'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/generator.rb:83:in `generate'
     # ./vendor/bundle/ruby/2.1.0/gems/rantly-1.1.0/lib/rantly/property.rb:31:in `check'
     # ./spec/features/store.rb:46:in `block (2 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # RuntimeError:
     #   HTTP error 409
     #   ./lib/moneta/adapters/couch.rb:53:in `store'
~~~

**Comment by asppsa** [comment]
~~~
irb(main):042:0> [Marshal.dump(2290273571719036605)].pack('m0')
=> "BAhsKwm9PsfYra/IHw=="
~~~

I believe this is another issue to do with slashes in Base64 strings

**Comment by asppsa** [comment]
Fixed in ff2b398340586d81c944f9f341711d732cfa595b
